kevm-js/kevm.js

```diff
@@ -1,74 +1,149 @@
-//const PRIVATEKEY = ''
-
-// IMPORTANT: after 'npm install' you must apply the following patch to node_modules/web3-eth-accounts/src/index.js
-// 179,182c179,182
-// <                 transaction.data,
-// <                 Bytes.fromNat(transaction.chainId || "0x1"),
-// <                 "0x",
-// <                 "0x"]);
-// —-
-// >                 transaction.data]);
-// >                 //Bytes.fromNat(transaction.chainId || "0x1"),
-// >                 //"0x",
-// >                 //"0x"]);
-// 187c187,188
-// <             var signature = Account.makeSigner(Nat.toNumber(transaction.chainId || "0x1") * 2 + 35)(Hash.keccak256(rlpEncoded), privateKey);
-// —-
-// >             //var signature = Account.makeSigner(Nat.toNumber(transaction.chainId || "0x1") * 2 + 35)(Hash.keccak256(rlpEncoded), privateKey);
-// >             var signature = Account.makeSigner(27)(Hash.keccak256(rlpEncoded), privateKey);
+/*
+FIRST: copy and run the following npm install lines into your console to install all all required modules (if not allready installed)
+
+npm install web3
+npm install crypto
```

Comment:
Btw, the npm page for `crypto` says that it's a deprecated package and should not be installed - https://www.npmjs.com/package/crypto

There's now the same built-in package, so I think the whole script should work as-is without installing it. I didn't install this package yet and script seem to work for me:

![](https://i.imgur.com/V9UkDkr.png)